build.gradle

```diff
@@ -312,7 +316,7 @@ pact {
     publish {
         pactDirectory = 'pacts'
         pactBrokerUrl = System.getenv("PACT_BROKER_FULL_URL") ?: 'http://localhost:80'
-        tags = [System.getenv("PACT_BRANCH_NAME") ?: 'Dev']
+        //tags = [System.getenv("PACT_BRANCH_NAME") ?: 'Dev']
```

Comment:
we can get rid of this tag as current broker does not support, we can add this back once we upgrade broker